pkg/kubelet/cri/remote/remote_image.go

```diff
@@ -226,11 +226,11 @@ func (r *remoteImageService) ImageFsInfo(ctx context.Context) ([]*runtimeapi.Fil
 	return r.imageFsInfoV1(ctx)
 }
 
-func (r *remoteImageService) imageFsInfoV1(ctx context.Context) ([]*runtimeapi.FilesystemUsage, error) {
+func (r *remoteImageService) imageFsInfoV1(ctx context.Context) (runtimeapi.ImageFsInfoResponse, error) {
 	resp, err := r.imageClient.ImageFsInfo(ctx, &runtimeapi.ImageFsInfoRequest{})
 	if err != nil {
 		klog.ErrorS(err, "ImageFsInfo from image service failed")
-		return nil, err
+		return runtimeapi.ImageFsInfoResponse{}, err
 	}
-	return resp.GetImageFilesystems(), nil
+	return *resp, nil
```

Comment:
ImageFsInfo is an exported function. Is there a way to preserve returning `[]*runtimeapi.FilesystemUsage`?

Reply:
I had this in the alternatives considered but it would require us to change the KEP design.

I was thinking if we want to just add container and images to the array, we should have a label that each container runtime writes into FilesystemUsage to say which filesystem this corresponds too.  

I didn't go this route because it seems a bit hacky and TBH I think returing the `ImageFsInfo` is probably the better long term move.  What happens if someone wants to add a new field, other than an array, to `ImageFsInfo`?   

The CRI API says that it returns a `ImageFSInfo` as the return type but we end up using this shortcut in kubelet.